src/gsCore/gsMultiPatch.h

```diff
@@ -168,6 +168,9 @@ class gsMultiPatch : public gsBoxTopology, public gsFunctionSet<T>
     /// manifold (ie. it has no boundaries)
     bool isClosed() { return this->nBoundary() == 0; }
 
+    /// \brief Returns true if gsMultiPatch is empty.
+    const bool isEmpty() const { return this->nPatches() <= 0; }
```

Comment:
As I think, this function is necessary, at least when we read files with gsReadFile.

Reply:
To conform to std, please rename it to "bool empty() const", and implement as { return m_patches.empty(); }